diffrax/_brownian/tree.py

```diff
@@ -222,31 +223,31 @@ def evaluate(
         t1: Optional[RealScalarLike] = None,
         left: bool = True,
         use_levy: bool = False,
-    ) -> Union[PyTree[Array], LevyVal]:
-        def _is_levy_val(obj):
-            return isinstance(obj, LevyVal)
+    ) -> Union[PyTree[Array], Union[TimeLevyArea, SpaceTimeLevyArea]]:
+        def _is_levy_area(obj):
+            return isinstance(obj, Union[TimeLevyArea, SpaceTimeLevyArea])
 
         t0 = eqxi.nondifferentiable(t0, name="t0")
         # map the interval [self.t0, self.t1] onto [0,1]
         t0 = linear_rescale(self.t0, t0, self.t1)
         levy_0 = self._evaluate(t0)
+        # Should this really be returning a levy area straight away?
```

Comment:
What do you mean?

Reply:
Apologies for the rogue comment, I have no idea what I was meaning either.